src/app/components/carpeta-paciente/carpeta-paciente.component.ts

```diff
@@ -129,36 +129,40 @@ export class CarpetaPacienteComponent implements OnInit {
 
     guardarCarpetaPaciente(nuevaCarpeta = false) {
         if (this.autorizado && this.nuevoNroCarpeta) {
-            this.carpetaPaciente.nroCarpeta = this.nuevoNroCarpeta.trim();
-            if (this.indiceCarpeta > -1) {
-                this.carpetaEfectores[this.indiceCarpeta] = this.carpetaPaciente;
+            if (/^\s*$/.test(this.nuevoNroCarpeta)) {
+                this.plex.info('warning', '', 'Ingrese un número de carpeta válido');
```

Comment:
Cuando hago click en guardar con un espacio en blanco, me aparece le modal correctamente. Pero paso seguido me saca de la edición de la carpeta. Debería quedarse en el formulario de alta. Eso se corrige con un return preventivo 
```suggestion
                this.plex.info('warning', '', 'Ingrese un número de carpeta válido');
                return;
```

Yo cambiaría el modal por un toas además.

Y a futuro lo mejor sería usar validación de formularios así el campo se marca en rojito pero no existe eso en esta pantalla todavía.